content/blog/2016/2016-07-04-ewm-alpha-version.adoc

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "GSoC: External Workspace Manager Plugin alpha version"
+tags:
+- pipeline
+- plugins
+- gsoc
+author: alexsomai
+---
+
+[NOTE]
+--
+This is a guest post by link:https://github.com/alexsomai[Alexandru Somai], Google Summer of Code student
+at Jenkins project.
+--
+
+An alpha version of the External Manager Plugin has been released!
+To be able to install this plugin, you must follow the steps from the Experimental Plugins Update Center
+link:/blog/2013/09/23/experimental-plugins-update-center/[blog post].
+
+The documentation that helps you get started with this plugin may be found on the
+link:https://github.com/jenkinsci/external-workspace-manager-plugin/blob/master/README.md[README] page.
+To get an idea of what this plugin does, which are the features implemented so far and to see a working demo of it,
+you can watch my mid-term presentation that is available link:https://youtu.be/u4zhxfUT8P4?t=22m7s[here].
+The slides for the presentation are shared on
+link:https://docs.google.com/presentation/d/1ZCYSIR2Tg466Ij1ghH5LSc8DLBCxWjIaD9IJcOyMZwU/edit?usp=sharing[Google Slides].
+
+I and my mentors, link:https://github.com/martinda[Martin] and link:https://github.com/oleg-nenashev[Oleg],
+have set-up public meetings related to this plugin. You are invited to join our discussions if you'd like to get
+more insight about the project. The meetings are taking place twice a week on the link:/hangout[Jenkins hangout],
+every Monday at
+link:http://www.timeanddate.com/worldclock/fixedtime.html?msg=External+Workspace+Manager+Plugin+(Mondays+weekly+recurring)&iso=20160606T12&p1=1440&ah=1[12 PM UTC]
+and every Thursday at
+link:http://www.timeanddate.com/worldclock/fixedtime.html?msg=External+Workspace+Manager+Plugin+(Thursdays+weekly+recurring)&iso=20160609T05&p1=1440&ah=1[5 PM UTC].
+
```

Comment:
Probably makes sense to say that these meetings will end when GSoC is concluded at the end of August 2016.